libcxx/test/std/atomics/atomics.ref/compare_exchange_strong.pass.cpp

```diff
@@ -24,79 +24,126 @@
 #include "test_helper.h"
 #include "test_macros.h"
 
-template <typename T>
+template <typename T, typename U>
+concept has_compare_exchange_strong_1 =
+    requires { std::declval<T>().compare_exchange_strong(std::declval<U&>(), std::declval<U>()); };
```

Comment:
Test failures indicated that inclusion of `<utility>` (providing `declval`) is missing. Ditto for `store.pass.hpp`.

But I wonder whether it would be better to use parameter list in requires-expressions to avoid `declval`.

Reply:
I followed your suggestion and got rid of `decval`